diff --git a/public/app/features/dashboard/components/DashboardSettings/SettingsCtrl.ts b/public/app/features/dashboard/components/DashboardSettings/SettingsCtrl.ts
old mode 100755
new mode 100644
diff --git a/public/app/features/folders/services/FolderPageLoader.ts b/public/app/features/folders/services/FolderPageLoader.ts
old mode 100755
new mode 100644
diff --git a/public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg b/public/app/plugins/datasource/elasticsearch/img/elasticsearch.svg
old mode 100755
new mode 100644
diff --git a/public/app/plugins/datasource/graphite/partials/query.editor.html b/public/app/plugins/datasource/graphite/partials/query.editor.html
old mode 100755
new mode 100644
diff --git a/public/app/plugins/datasource/stackdriver/partials/query.editor.html b/public/app/plugins/datasource/stackdriver/partials/query.editor.html
old mode 100755
new mode 100644
diff --git a/public/app/plugins/panel/graph/graph.ts b/public/app/plugins/panel/graph/graph.ts
old mode 100755
new mode 100644
diff --git a/public/fonts/grafana-icons.svg b/public/fonts/grafana-icons.svg
old mode 100755
new mode 100644
diff --git a/public/sass/components/_panel_text.scss b/public/sass/components/_panel_text.scss
index e69de29bb..cd0183132 100644
--- a/public/sass/components/_panel_text.scss
+++ b/public/sass/components/_panel_text.scss
@@ -0,0 +1 @@
+// nothing here
