diff --git a/pkg/tests/api/plugins/api_plugins_test.go b/pkg/tests/api/plugins/api_plugins_test.go
index 04073f43e1..c622391120 100644
--- a/pkg/tests/api/plugins/api_plugins_test.go
+++ b/pkg/tests/api/plugins/api_plugins_test.go
@@ -58,6 +58,7 @@ func TestPlugins(t *testing.T) {
 		})
 
 		t.Run("Request is not forbidden if from an admin", func(t *testing.T) {
+			t.Skip("this test requires connectivity to https://grafana.com/api/plugins/repo/test")
 			statusCode, body := makePostRequest(t, grafanaAPIURL(usernameAdmin, grafanaListedAddr, "plugins/test/install"))
 
 			assert.Equal(t, 404, statusCode)
